test/Microsoft.NET.TestFramework/Commands/AddReferenceCommand.cs

```diff
@@ -17,7 +17,7 @@ public override CommandResult Execute(IEnumerable<string> args)
         {
             List<string> newArgs = new();
             newArgs.Add("add");
-            if (!string.IsNullOrEmpty(_projectName))
+            if (_projectName is not null && !string.IsNullOrEmpty(_projectName))
```

Comment:
We've updated the build to no longer warn for the .NET Framework compilation for `string.IsNullOrEmpty` so this added check is no longer necessary. See: https://github.com/dotnet/sdk/issues/45299